res/database/Default/database_programmes.xml

```diff
@@ -1301,15 +1364,17 @@ Berührende, einigermaßen glaubwürdige Geschichte und eingängige Musik von Ha
 		<programme id="fc0bf277-6d13-48ea-8829-f5f0e1ac72e1" product="1" licence_type="1" tmdb_id="0" imdb_id="tt0099348" creator="">
 			<title>
 				<de>Der mit den Wölfen schunkelt</de>
-				<en></en>
+				<en>Who Sways with the Wolves</en>
 			</title>
 			<title_original>
 				<de>Der mit dem Wolf tanzt</de>
+				<en>Dances with Wolves</en>
 			</title_original>
 			<description>
 				<de>Ein amerikanischer Soldat freundet sich mit einem Indianerstamm an.
 Sehenswertes Regie-Debüt von [0|Full]</de>
-				<en></en>
+				<en>An American soldier befriends an Indian tribe.
+Worth seeing directorial debut of [0|Full].</en>
```

Comment:
"worth seeing..." - maybe there could be a smoother less word by word translation